bot/exts/moderation/verification.py

```diff
@@ -834,19 +834,21 @@ async def bot_check(ctx: Context) -> bool:
 
     @command(name='verify')
     @has_any_role(*constants.MODERATION_ROLES)
-    async def apply_developer_role(self, ctx: Context, user: discord.Member) -> None:
-        """Command for moderators to apply the Developer role to any user."""
+    async def perform_manual_verification(self, ctx: Context, user: discord.Member) -> None:
+        """Command for moderators to verify any user."""
         log.trace(f'verify command called by {ctx.author} for {user.id}.')
-        developer_role = self.bot.get_guild(constants.Guild.id).get_role(constants.Roles.verified)
 
-        if developer_role in user.roles:
-            log.trace(f'{user.id} is already a developer, aborting.')
-            await ctx.send(f'{constants.Emojis.cross_mark} {user} is already a developer.')
+        if user.pending:
```

Comment:
When user have pending True, I think this means that user isn't already verified?

Reply:
I think you're right--this should probably be `if not user.pending`.